Pickle-Finance-Agents/RangeManagementMonitor/README.md

```diff
@@ -0,0 +1,22 @@
+# Pickle Range Management Monitor
+
+## Description
+
+Monitors the strategies to verify they are in range and earning fees.
+
+## Supported Chains
+
+- Ethereum
```

Comment:
Should we mention Ethereum? The keeper doesn't exist on ethereum.

Reply:
Oh you mean that the current implementation doesn't work on mainnet, that is right. I added Ethereum because the agent will work on mainnet too if the params are changed accordingly (thinking that they want the agent in mainnet later).

I think it should be removed as you suggest, because in the other way we can list any compatible chain 🙃